specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2344 @@
+                    "description": "Text content of the cell.",
+                    "type": "string"
+                },
+                "boundingBox": {
+                    "description": "Bounding box of the cell.",
+                    "$ref": "#/definitions/BoundingBox"
+                },
+                "confidence": {
+                    "description": "Confidence value.",
+                    "$ref": "#/definitions/Confidence"
+                },
+                "elements": {
+                    "description": "When includeTextDetails is set to true, a list of references to the text elements constituting this table cell.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ElementReference"
+                    }
+                },
+                "isHeader": {
+                    "description": "Is the current cell a header cell?",
+                    "type": "boolean",
+                    "default": false,
+                    "x-nullable": false
+                },
+                "isFooter": {
+                    "description": "Is the current cell a footer cell?",
+                    "type": "boolean",
+                    "default": false,
+                    "x-nullable": false
+                }
+            }
+        },
+        "DocumentResult": {
+            "description": "A set of extracted fields corresponding to the input document.",
+            "type": "object",
+            "required": [
+                "docType",
+                "pageRange",
+                "fields"
+            ],
+            "properties": {
+                "docType": {
+                    "description": "Document type.",
+                    "type": "string"
+                },
+                "modelId": {
+                    "description": "Model identifier.",
+                    "type": "string",
+                    "format": "uuid",
+                    "x-nullable": false
+                },
+                "pageRange": {
+                    "description": "First and last page number where the document is found.",
+                    "type": "array",
+                    "minItems": 2,
+                    "maxItems": 2,
+                    "items": {
+                        "type": "integer",
+                        "minimum": 1,
+                        "x-nullable": false
+                    }
+                },
+                "docTypeConfidence": {
+                    "description": "Predicted document type confidence.",
+                    "$ref": "#/definitions/Confidence"
+                },
+                "fields": {
+                    "description": "Dictionary of named field values.",
+                    "type": "object",
+                    "additionalProperties": {
+                        "$ref": "#/definitions/FieldValue"
+                    }
+                }
+            }
+        },
+        "FieldValue": {
```

Comment:
needs properties "valueGender" and "valueCountry"

Reply:
@ramparab Can you please include the following?

```
                "valueGender": {
                    "description": "Gender value: M, F, or X.",
                    "type": "string",
                    "enum": [
                        "M",
                        "F",
                        "X"
                    ],
                    "x-nullable": false
                },
                "valueCountry": {
                    "description": "3-letter country code (ISO 3166-1 alpha-3).",
                    "type": "string",
                    "x-nullable": false
                },
```

Reply:
@ChrisHMSFT 
 Is there a standard for the Cognitive services on which format they should use for country code? 
[Text Analytics](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/TextAnalytics.json#L1118) uses 2-letter country code `ISO 639-1`

Reply:
done

Reply:
I don't want to block the PR because of this since it is a preview. I will keep following up offline

Reply:
After some emails, I learned that 3166 is part of BCP-47 which is MS guideline. 
According to needs, the service team can use different codes. TA is focused on Language, while FR is focused on Country/Region, so each is using the respective standard.